### ISSUE TITLE
Display remaining message to read

### ISSUE THREAD

**Issue by blacoste** [siacs/Conversations#3662]
New feature: 
Display how many messages are still to be read, next or within the existing bottom-right arrow that allows the user to jump to the latest message. Number would be updated upon scrolling.

For instance when reading MUCs I may be missing a lot of messages and start to read them. Would give me an indication of my progression and if I give up or not.

**Comment by licaon-kter** [comment]
Already available, close as invalid.

**Comment by blacoste** [comment]
It's not displaying anything on 2.7.1+fcr.

**Comment by licaon-kter** [comment]
Right, indeed....wasn't this a feature before or did I imagine it?

**Comment by ltGuillaume** [comment]
@licaon-kter I do see message counts in green at times, but those are the newly received messages _while you're in the conversation_. Perhaps that's what confused you?
